apps/dashboard/app/views/files/_inline_js.html.erb

```diff
@@ -1,13 +0,0 @@
-<script>
-const dashboard_files_directory_download_error_modal_title = '<%= t('dashboard.files_directory_download_error_modal_title') %>';
-const alert = '<%= alert %>';
```

Comment:
I cannot find where these 2 configurations are used and so haven't ported them. I don't believe they're used anymore.